src/site/content/en/blog/flexbox-gap/index.md

```diff
@@ -0,0 +1,222 @@
+{% endAside %}
+
+`gap` is [flow relative](https://www.w3.org/TR/css-logical-1/#intro), meaning it changes dynamically based on the direction of content flow. For example, `gap` will automatically adjust for the different`writing-mode` or `direction` values that you set for your international users. This significantly eases the burden of spacing challenges for the component and CSS author. **Less code scaling further.**
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <!-- <source src="https://storage.googleapis.com/web-dev-assets/portals_vp9.webm" type="video/webm; codecs=vp8"> -->
+    <source src="https://storage.googleapis.com/web-dev-assets/flexbox-gap/gap-i18n.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Gap demonstrating localization support, as it handles changes to direction and writing-mode: <a href="https://codepen.io/argyleink/pen/MWaoZJM">Codepen</a> | <a href="https://twitter.com/argyleink/status/1254794309263491072?s=20">Tweet</a>
+  </figcaption>
+</figure>
+
+<!-- [TODO:? glitch embed of this https://codepen.io/argyleink/pen/MWaoZJM] -->
+
+### Usage
+`gap` accepts any CSS [length](https://drafts.csswg.org/css-values-4/#lengths) or [percentage](https://www.w3.org/TR/css-values-3/#percentages) as a value. 
+
+```css
+.gap-example {
+  display: grid;
+  gap: 10px;
+  gap: 2ch;
+  gap: 5%;
+  gap: 1em;
+  gap: 3vmax;
+}
+```
+
+<br>
+
+Gap can be passed 1 length, which will be used for both row and column.
+
+<div class="w-columns">
+{% Compare 'better', 'Shorthand' %}
+```css
+.grid {
+  display: grid;
+  gap: 10px;
+}
+```
+
+{% CompareCaption %}
+Set both rows and columns at once
+{% endCompareCaption %}
+
+{% endCompare %}
+
+{% Compare 'better', 'Expanded' %}
+```css
+.grid {
+  display: grid;
+  row-gap: 10px;
+  column-gap: 10px;
+}
+```
+
+{% CompareCaption %}
+Set them separately
+{% endCompareCaption %}
+
+{% endCompare %}
+</div>
+
+<br>
+
+Gap can be passed 2 lengths, which will be used for row and column.
+
+<div class="w-columns">
+{% Compare 'better', 'Shorthand' %}
+```css
+.grid {
+  display: grid;
+  gap: 10px 5%;
+}
+```
+
+{% CompareCaption %}
+Set both rows and columns **separately** at once
+{% endCompareCaption %}
+
+{% endCompare %}
+
+{% Compare 'better', 'Expanded' %}
+```css
+.grid {
+  display: grid;
+  row-gap: 10px;
+  column-gap: 5%;
+}
+```
+
+{% endCompare %}
+</div>
+
+
+## Flexbox `gap`
+
+All of that `gap` goodness we just reviewed is available with Flexbox containers now! Before `gap` was in Flexbox, strategies involved negative margins, complex selectors, `:last` or `:first` type pseudo-class selectors, or other means to manage the space of a dynamically layed-out and wrapping set of children.
+
+### Previous Attempts
+
+Example #1 - pseudo-class selectors
+```css
+.layout > :not(:first-child) {
+  margin-block-end: 10px;
+  margin-inline-end: 5%;
+}
+```
```

Comment:
How about styling this with the `{% Compare 'worse' %}` component? You can give it a custom label if you want to use a gentler word. https://web.dev/handbook/web-dev-components/#compare-with-custom-labels

Reply:
done 👍